VocabularyTrainer/Home Screen/AboutViewController.swift

```diff
@@ -0,0 +1,194 @@
+    private let bottomView: UIView = {
+        let view = UIView()
+        view.backgroundColor = HomeViewModel.Colors.flippyGreen
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    // MARK: - Init
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        webView.navigationDelegate = self
+        setupUI()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    // MARK: - View Lifecycle
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        navigateToPP()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        title = Strings.title
+        view.backgroundColor = .systemBackground
+        navigationItem.leftBarButtonItem = closeButton
+        webView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(webView)
+        view.addSubview(versionLabel)
+        view.addSubview(bottomView)
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            webView.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor),
+            webView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor),
+            webView.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor),
+            webView.bottomAnchor.constraint(equalTo: versionLabel.topAnchor),
+
+            versionLabel.leftAnchor.constraint(equalTo: view.leftAnchor),
+            versionLabel.rightAnchor.constraint(equalTo: view.rightAnchor),
+            versionLabel.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor),
+
+            bottomView.topAnchor.constraint(equalTo: versionLabel.bottomAnchor),
+            bottomView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            bottomView.rightAnchor.constraint(equalTo: view.rightAnchor),
+            bottomView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    /// Navigates to initial initial website (`urlString`).
+    private func navigateToPP() {
+        if let ppUrl = URL(string: urlString) {
+            webView.load(URLRequest(url: ppUrl))
+        } else {
+            loadLocalPp()
+        }
+    }
+
+    /// Loads fallback local html.
+    private func loadLocalPp() {
+        guard let htmlFile = Bundle.main.path(forResource: "pp", ofType: "html") else { return }
+        guard let html = try? String(contentsOfFile: htmlFile, encoding: String.Encoding.utf8) else { return }
+        webView.loadHTMLString(html, baseURL: nil)
+    }
+
+    /// Creates and shows loading alert.
+    private func showLoadingIndicator() {
+        loadingController = UIAlertController(title: nil,
+                                              message: Strings.pleaseWaitTitle,
+                                              preferredStyle: .alert)
+        if let loadingController = loadingController {
```

Comment:
Maybe create that in a separated method

Reply:
makes sense, moved it to where the property is initialized (see a19a2958c771f844d9f71ab443d46203e76011fd)

not sure why this was an optional before. I just copied it from my old view controller code 😄